intelmq/bots/experts/modify/examples/morefeeds.conf

```diff
@@ -226,6 +226,15 @@
         }
     },
     {
+        "rulename": "ramnit",
+        "if": {
+            "malware.name": "^ramnit-0x.*$"
```

Comment:
Yes. Sample malware names from recent botnet drone feeds:
ramnit-0x4ec815e2
ramnit-0x606d35bf

So a more strict rule would be "^ramnit-0x[0-9a-f]{8}$" but I don't know if this is really necessary and it will probably fail to catch malware names if the length of the hex value changes.